pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md

```diff
@@ -0,0 +1,1278 @@
+
+    ```shell
+    git clone https://github.com/ovh/docs.git
+
+    cd docs/pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/
+    ```
+
+2. Next, add the `TrilioVault` Helm repository, and list the available charts:
+
+    ```shell
+    helm repo add triliovault-operator http://charts.k8strilio.net/trilio-stable/k8s-triliovault-operator
+    helm repo add triliovault http://charts.k8strilio.net/trilio-stable/k8s-triliovault
+    helm search repo triliovault-operator
+    ```
+
+    The output looks similar to the following:
+
+    ```text
+    NAME                                            CHART VERSION   APP VERSION     DESCRIPTION
+    triliovault-operator/k8s-triliovault-operator   2.6.4           2.6.4           K8s-TrilioVault-Operator is an operator designe...
+    triliovault/k8s-triliovault-operator            0.9.0           0.9.0           K8s-TrilioVault-Operator is an operator designe...
+    ```
+    
+3. The chart of interest is `triliovault-operator/k8s-triliovault-operator`, which will install `TrilioVault for Kubernetes Operator` on the cluster. You can run `helm install` command to install the Operator. Install `TrilioVault for Kubernetes Operator` using `Helm`:
+
+    ```shell
+    TRILIOVAULT_CHART_VERSION="2.6.4"
+
+    helm install triliovault-operator triliovault-operator/k8s-triliovault-operator --version "${TRILIOVAULT_CHART_VERSION}" --namespace tvk --create-namespace
+    ```
+
+    **Note:**
+
+    A `specific` version for the `TrilioVault-Operator` Helm chart is used. In this case `2.6.4` is picked, which maps to the `2.6.4` version of the application (see the output from `Step 2.`). It’s good practice in general, to lock on a specific version. This helps to have predictable results, and allows versioning control via `Git`.
+
+Now, please check your `TVK` deployment:
+
+```shell
+helm ls -n tvk
+```
+
+The output looks similar to the following (`STATUS` column should display `deployed`):
+
+```text
+NAME                    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                           APP VERSION
+triliovault-operator    tvk             1               2021-12-21 08:13:41.644264863 +0000 UTC deployed        k8s-triliovault-operator-2.6.4  2.6.4
+```
+
+Next, verify that `TrilioVault-Operator` is up and running:
+
+```shell
+kubectl get deployments -n tvk
+```
+
+The output looks similar to the following (deployment pods must be in the `Ready` state):
+
+```text
+NAME                                		READY   UP-TO-DATE   AVAILABLE   AGE
+triliovault-operator-k8s-triliovault-operator   1/1     1            1           3d
+```
+
+Now, you can install `TrilioVault Manager Application` using yaml definition (pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/assets/manifests/triliovault-manager-v2.6.4.yaml)
```

Comment:
There are many things I don't get :)
First, you may add a warning here because the version 2.6.4 is hard coded inside the manifest, so if the version differs, you'll get an error.
Then, when I execute the `kubectl apply` command with this file, it retruns:
`denied the request: an instance of TrilioVaultManager 'triliovault-manager' already exits with 'Cluster' ApplicatioScope in 'tvk' namespace`
and at this moment, the command `helm ls -n tvk` returns:
```
NAME                   	NAMESPACE	REVISION	UPDATED                                	STATUS  	CHART                         	APP VERSION
triliovault-manager-tvk	tvk      	1       	2022-01-14 09:51:47.285819484 +0000 UTC	deployed	k8s-triliovault-2.6.6         	2.6.6      
triliovault-operator   	tvk      	1       	2022-01-14 10:50:42.007746 +0100 CET   	deployed	k8s-triliovault-operator-2.6.6	2.6.6   
```
Maybe something new in the deployment for the 2.6.6 version?

Reply:
With the latest release v2.6.5 and 2.6.6, we have removed the dependency from the yaml manifest to install the `triliovault-manager`. I have revamped the installation section which should address this problem.